lib/resource_kit/testing/action_handler_matchers.rb

```diff
@@ -21,16 +21,29 @@ def with(options, &block)
       def matches?(subject, &block)
         @handled_block ||= block
         action = subject.resources.find_action(self.action)
-        return false unless action
+        unless action
+          @failures = "expected :#{self.action} to be handled by the class."
```

Comment:
I was trying to maintain homogeneity between the two matchers, but you are totally right.